admin_app/src/app/urgency-rules/page.tsx

```diff
@@ -142,160 +139,219 @@ const UrgencyRulesPage = () => {
+                          marginBottom: 1,
+                        }}
+                        InputProps={{
+                          style: { backgroundColor: "white" },
+                          endAdornment: (
+                            <InputAdornment position="end" sx={{ paddingRight: 2 }}>
+                              <IconButton
+                                onMouseDown={() => {
+                                  addOrUpdateItem(index);
+                                  setEditableIndex(-1);
+                                }}
+                                edge="end"
+                              >
+                                <Send fontSize="small" />
+                              </IconButton>
+                            </InputAdornment>
+                          ),
+                        }}
+                      />
+                    ) : (
+                      <ListItemText
+                        id={`checkbox-list-label-${index}`}
+                        primary={urgencyRule.urgency_rule_text}
+                        secondary={
+                          urgencyRule.updated_datetime_utc ? (
+                            "Last updated: " +
+                            new Date(urgencyRule.updated_datetime_utc).toLocaleString(
+                              undefined,
+                              {
+                                day: "numeric",
+                                month: "numeric",
+                                year: "numeric",
+                                hour: "numeric",
+                                minute: "numeric",
+                                hour12: true,
+                              },
+                            )
+                          ) : (
+                            <span>
+                              <Layout.Spacer multiplier={0.8} />
+                              <LinearProgress color="secondary" />
+                            </span>
+                          )
+                        }
+                        sx={{
+                          paddingTop: 0.3,
+                          paddingBottom: 0.3,
+                          paddingRight: 12,
+                          ".MuiListItemText-secondary": {
+                            fontSize: "0.75rem",
+                          },
+                        }}
+                      />
+                    )}
+                  </ListItem>
+                );
+              })}
+            </List>
+          </Box>
+        </Layout.FlexBox>
+        {!openSidebar && (
+          <div style={{ position: "relative" }}>
+            <Fab
+              variant="extended"
+              sx={{
+                bgcolor: "orange",
+                position: "absolute",
+                bottom: 80,
+                right: 150,
+              }}
+              onClick={handleSidebarToggle}
+            >
+              <PlayArrow />
+              <Layout.Spacer horizontal multiplier={0.3} />
```

Comment:
The positioning of this button looks odd.

<img width="1579" alt="image" src="https://github.com/user-attachments/assets/9f415264-3c2b-4f32-bf0a-75b3c792efa9">

Would be nice to have it at the bottom of ths screen but aligned under the "New" button.